src/main/java/frc/team2412/robot/commands/intake/IntakeRetractCommand.java

```diff
@@ -0,0 +1,21 @@
+package frc.team2412.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.IntakeSubsystem;
+
+public class IntakeRetractCommand extends CommandBase{
+    private final IntakeSubsystem subsystem;
+
+    public IntakeRetractCommand(IntakeSubsystem subsystem) {
+        this.subsystem = subsystem;
+        addRequirements(subsystem);
+
+    }
+
+    @Override
+    public void execute() {
+        subsystem.intakeRetract();
+        
+    }
+}
```

Comment:
It would be good to make sure this command (have isFinished return true as mentioned above) ends since it just toggles the solenoid.